website/content/docs/secrets/pki/cmpv2.mdx

```diff
@@ -0,0 +1,130 @@
+---
+layout: docs
+page_title: Certificate Management Protocol v2 (CMPv2) within Vault | PKI - Secrets Engines
+description: An overview of the Enrollment over Secure Transport protocol implementation within Vault.
+---
+
+# PKI secrets engine - Certificate Management Protocol v2 (CMPv2) <EnterpriseAlert inline="true" />
+
+This document covers configuration and limitations of Vault's PKI Secrets Engine
+implementation of the [CMPv2 protocol](https://datatracker.ietf.org/doc/html/rfc4210) <EnterpriseAlert inline="true" />.
+
+## What is Certificate Management Protocol v2 (CMPv2)?
+
+The CMP protocol is an IETF standardized protocol, [RFC 4210](https://datatracker.ietf.org/doc/html/rfc4210),
+that allows clients to acquire client certificates and their associated Certificate
+Authority (CA) certficates.
+
+## Enabling CMPv2 support on a Vault PKI  mount
+
+To configure an existing mount to serve EST clients, the following steps are
+required, which are broken down into three main categories:
+
+ 1. [Authentication mechanisms](#configuring-est-authentication)
+ 1. [Updating PKI tunable parameters](#updating-the-pki-mount-tunable-parameters)
+ 1. [PKI CMPv2 configuration](#pki-cmpv2-configuration)
+
+### Configuring CMPv2 Authentication
+
+At this time, Vault's implementation of CMPv2 supports only 
+[Certificate TLS authentication](/vault/docs/auth/cert), where clients proof
+of posession of a TLS client certificate authenticates them to Vault.
+
+Authentication leverages a separate Vault authentication
+mount, within the same namespace, to validate the client provided credentials
+along with the client's ACL policy to enforce. 
+
+For proper accounting, a mount supporting CMPv2 authentication should be
+dedicated to this purpose, not shared with other workflows.  In other words,
+create a new certificate auth mount for CMPv2 even if you already have one 
+another in use for other purposes.
+
+When setting up the authentication mount for CMPv2 clients, the token type must
+be configured to return [batch tokens](/vault/docs/concepts/tokens#batch-tokens).
+Batch tokens are required to avoid an excessive amount of leases being generated
+and persisted as every CMPv2 incoming request needs to be authenticated.
```

Comment:
```suggestion

- To ensure proper accounting and auditing, do not share the CMPv2 mount with
  other workflows. Always create a dedicated certificate authentication mount
 for CMPv2.
- Configure the token type for your CMPv2 authentication mount to return
  [batch tokens](/vault/docs/concepts/tokens#batch-tokens) to avoid excessive
  lease generation and persistance.
```

This feels like best practice recommendations rather than education (overview)
or instructional (how-to) information. If we have enough recommendations to
justify a separate page, I recommend moving it to a dedicated document. But if
we only have a few recs, I'd recommend creating a dedicated "Best practices"
section in the overview an collecting the recommendations there. (I've styled the
suggestion assuming the latter).

Style correction: write in active voice.